<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>1280</width>
        <height>720</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #F4F1DE;</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="home_page">
       <property name="minimumSize">
        <size>
         <width>1280</width>
         <height>600</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="top_frame_3">
          <property name="minimumSize">
           <size>
            <width>1300</width>
            <height>440</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_lesson">
             <property name="maximumSize">
              <size>
               <width>350</width>
               <height>280</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
border: 5px solid #E07A5F;
border-radius: 20px;
}

QPushButton:hover:!pressed {
background-color: #3D405B;
color: #F4F1DE;
}

QPushButton:pressed {
background-color: #F4F1DE;
color: #E07A5F;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Images/lessons_icon.png</normaloff>../Images/lessons_icon.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>400</width>
               <height>400</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="bottom_frame_3">
          <property name="minimumSize">
           <size>
            <width>1301</width>
            <height>151</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>1280</width>
               <height>120</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>../Images/banner_icon.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="lesson_page">
       <property name="minimumSize">
        <size>
         <width>1280</width>
         <height>720</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="top_frame_2">
          <property name="minimumSize">
           <size>
            <width>1280</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #E07A5F;
color: #3D405B; </string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="btn_back_to_home_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>60</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>4</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="mouseTracking">
              <bool>true</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	background-color: ;
	border: none;
}

QPushButton:hover:!pressed
{
	background-color: #F4F1DE;
	border-radius: 30px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Images/back.png</normaloff>../Images/back.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>250</width>
               <height>250</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="default">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_lesson_headline">
             <property name="font">
              <font>
               <family>Kalpurush</family>
               <pointsize>30</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
padding-left: 5px;
font: 30pt &quot;Kalpurush&quot;;</string>
             </property>
             <property name="text">
              <string>নাম শিখন - পাঠ ১</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="bottom_frame_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>550</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(244, 241, 222);
border: none;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="btn_prev_lesson">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>650</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
border: 3px solid #E07A5F;
border-radius: 20px;
}

QPushButton:hover:!pressed {
background-color: #3D405B;
color: #F4F1DE;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Images/backward.png</normaloff>../Images/backward.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_image">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>800</width>
               <height>550</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>30</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 3px dashed rgb(222, 219, 202);
color:  rgb(61, 64, 91);</string>
             </property>
             <property name="text">
              <string>ছবি</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_image_text">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>500</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>700</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>25</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">border: 3px dashed rgb(222, 219, 202);
color:  rgb(61, 64, 91);</string>
             </property>
             <property name="text">
              <string>পরিচয়</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_next_lesson">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>650</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
border: 3px solid #E07A5F;
border-radius: 20px;
}

QPushButton:hover:!pressed {
background-color: #3D405B;
color: #F4F1DE;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>../Images/forward.png</normaloff>../Images/forward.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>100</width>
               <height>100</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
